pypesto/hierarchical/optimal_scaling_approach/optimal_scaling_parameter.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+from ...C import InnerParameterType
+from ..parameter import InnerParameter
+
+logger = logging.getLogger(__name__)
+
+
+class OptimalScalingParameter(InnerParameter):
+    """Inner parameter of the optimal scaling hierarchical optimization problem.
+
+    Attributes
+    ----------
+    category:
+        Category index.
+    group:
+        Group index.
+    value:
+        Current value of the inner parameter.
+    estimate:
+        Whether to estimate inner parameter in inner subproblem.
+    """
+
+    def __init__(
+        self,
+        *args,
```

Comment:
are the *args, **kwargs arguments needed here? If the original parameters need to be passed (such as id, lb, ub, dumy_value), I think it is better design to keep them here explicitly, as *args, **kwargs may hide stuff from the developer.